utilities/dataflow_hcapi_to_gcs_pubsub/requirements.txt

```diff
@@ -0,0 +1,8 @@
+logging
+apache-beam[gcp]==2.48.0
```

Comment:
We can use the most recent version (2.53.0)

Reply:
We can but I have tested on this version so have kept this as is for now. I don't know if it's going to break the code without testing.